### PR TITLE
Add CI test - first step

### PR DESCRIPTION
**Pull request by jano-wol**

### CODE REVIEW

File: src/scripts/test.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+SCRIPT_FOLDER=$(dirname "${0}") 
+WORKSPACE_FOLDER=$(readlink -e "${SCRIPT_FOLDER}/../")
```
Comment (anwolosz) [inline src/scripts/test.sh:3]: "${SCRIPT_FOLDER}/../**../**"

Comment (anwolosz) [inline src/scripts/test.sh:3]: ${SCRIPT_FOLDER}/../
One more "../" needed.
${SCRIPT_FOLDER}/../../

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+
+    - name: Run tests
+      run: |
+        ./src/scripts/test.sh release
```
Comment (anwolosz) [inline .github/workflows/ci.yml:19]: "release" not needed

File: src/scripts/test.sh (hunk 1 above)
Comment (jano-wol) [reply src/scripts/test.sh]: Fixed

File: .github/workflows/ci.yml (hunk 2 above)
Comment (jano-wol) [reply .github/workflows/ci.yml]: Fixed

**Pull request merged**